client/src/utils/Drag.ts

```diff
@@ -799,7 +808,7 @@ const drop = () => {
       const gridChildren = dragElement.parentElement?.parentElement?.children;
       const dragrect = dragElement.children[0].getBoundingClientRect();
 
-      if (gridChildren && dragTarget) {
+      if (gridChildren !== undefined && gridChildren.length > 1) {
```

Comment:
We risking type error but winnable?
```suggestion
      if (gridChildren?.length > 1) {
```

Also, is `gridChildren` always undefined? (i suspect it could be `null`)

Reply:
keeping as original as it handle undefined cases.